jujugui/static/gui/src/app/models/legacy-handlers.js

```diff
@@ -299,9 +318,22 @@ YUI.add('juju-legacy-delta-handlers', function(Y) {
       @return {undefined} Nothing.
      */
     annotationInfo: function(db, action, change) {
+      var cleanUpEntityTags = function(tag) {
+        var result = tag.replace(/^(service|unit|machine|environment)-/, '');
```

Comment:
Same as above. The two functions are slightly different, and this one is inlined because it's only intended to be use in the legacy API ghetto.